Spanner/src/SpannerClient.php

```diff
@@ -73,6 +73,26 @@
  * $spanner = new SpannerClient();
  * ```
  *
+ * ```
+ * use Google\Cloud\Spanner\SpannerClient;
+ * use Google\Cloud\Spanner\V1\DirectedReadOptions\ReplicaSelection\Type as ReplicaType;
+ *
+ * $spanner = new SpanneClient();
+ * $directedOptions = [
```

Comment:
This example looks incorrect?
Is the intention here to create `$directedOptions` first and then pass it in client creation?